From b6f6a4b9e8dd509f7c6a91840189867812b0294e Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 13 Feb 2025 15:02:40 +0100
Subject: [PATCH 1504/1695] plat-stm32mp1: syscfg: set
 SYSCFG_CMPCR_READY_TIMEOUT_US to 10ms

CHange timeout to 10 ms instead of 1 ms.
On stm32mp13 we measure 1.5ms delay to have CMPCR_READY equal to 1.
Use 10 ms to be aligned with TF-A timeout.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1084f270e90691bca100f2d6040a29ee0d22a155 plat-stm32mp1: syscfg: set SYSCFG_CMPCR_READY_TIMEOUT_US to 10ms]
Change-Id: I123b45c9db2777272a2d60a9a6788d104f1b52f3
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/437416
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c
index e3c78fef9..675b0d3a0 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c
@@ -59,7 +59,7 @@
 #define SYSCFG_CMPCR_RAPSRC			GENMASK_32(23, 20)
 #define SYSCFG_CMPCR_ANSRC_SHIFT		U(24)
 
-#define SYSCFG_CMPCR_READY_TIMEOUT_US		U(1000)
+#define SYSCFG_CMPCR_READY_TIMEOUT_US		U(10000)
 
 #define CMPENSETR_OFFSET			U(0x4)
 #define CMPENCLRR_OFFSET			U(0x8)
-- 
2.47.2

